Buildings/Fluid/Humidifiers/DXDehumidifier.mo

```diff
@@ -0,0 +1,289 @@
+within Buildings.Fluid.Humidifiers;
+model DXDehumidifier "DX dehumidifier"
+  extends Buildings.Fluid.Interfaces.PartialTwoPort;
+
+  parameter Modelica.Units.SI.VolumeFlowRate V_flow_nominal(min=0) = 5.805556e-7
+    "Rated water removal rate, in m3/s"
+    annotation (Dialog(group="Nominal condition"));
+
+  parameter Boolean addPowerToMedium = true
+    "Transfer power and heat to the fluid medium";
+
+  parameter Real eneFac_nominal(
+    final min=0) = 3.412
+    "Rated energy factor, in L/kwh"
+    annotation (Dialog(group="Nominal condition"));
+
+  parameter Buildings.Fluid.Humidifiers.Data.Generic per
+    "Performance data"
+    annotation (choicesAllMatching=true,
```

Comment:
make sure the parameters in the same group consequently together. 

For example here, the parameter `V_flow_nominal` is in group `Nominal condition`, the parameter `addPowerToMedium` is not grouped, and then again, the parameter `eneFac_nominal` is in group `Nominal condition` again. It should be avoided.